client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -205,8 +249,16 @@ class Upload extends Component {
 		tryandcustomize.action( this.props.themeId );
 	};
 
+	onUpsellNudgeClick = () => {
+		const isEligibleForHostingTrial = this.props.isEligibleForHostingTrial;
```

Comment:
That's a funny assignment. We're not destructuring nor renaming the var 🤔 What was the intent here?